Kernel/System/CustomerUser/DB.pm

```diff
@@ -16,8 +16,6 @@ use Kernel::System::VariableCheck qw(:all);
 use Crypt::PasswdMD5 qw(unix_md5_crypt apache_md5_crypt);
 use Digest::SHA;
 
-use Kernel::System::VariableCheck qw(:all);
```

Comment:
@reneeb IsArrayRefWithData and IsHashRefWithData are used in the code.

Reply:
That's why the "use" in line 14 is stll there ;-)